ttnn/cpp/ttnn/operations/conv/conv2d/conv2d_utils.cpp

```diff
@@ -154,6 +114,11 @@ ParallelConfig determine_parallel_config(
     } else if (shard_layout == TensorMemoryLayout::BLOCK_SHARDED) {
         uint32_t start_divisor =
                 block_shard_orientation == ShardOrientation::COL_MAJOR ? compute_grid_size.x : compute_grid_size.y;
+        if (is_non_tile_mul_shard_width) {
+            out_nhw_ntiles = tt::round_up(batch_size * output_height * output_width, tt::constants::TILE_HEIGHT);
```

Comment:
can we just resue result out_nhw_ntiles
I'm not following why in this case why we are not dividing with tile height?
Name of the variable suggests that it should be height in tile size?